tests/setup_cfg_fmt_test.py

```diff
@@ -454,7 +461,8 @@ def test_guess_python_requires_tox_ini_dashed_name(tmpdir):
         'version = 1.0\n',
     )
 
-    main((str(setup_cfg), '--min-py3-version=3.4', '--max-py-version=3.7'))
+    r = main((str(setup_cfg), '--min-py3-version=3.4', '--max-py-version=3.7'))
+    assert r
```

Comment:
prefered introducing a local variable then having to split the invocation over multiple lines 🤔 (line length limit would be triggered otherwise)

Reply:
does ret fit?

Reply:
Sometimes, not in general.

Reply:
maybe this avoids the 1-letter variable name?

```python
args = (...)
assert main(args)
```

Reply:
Done.